fix: QoL&Bug fixes

larentoun opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
<!-- Include a small to medium description of what your PR changes. Document every changes or this may delay review or even discourage maintainers from merging your PR! -->
Читать Changelog

Фиксит из #баг-репорты:
[Дюпаем драконов](https://discord.com/channels/617003227182792704/1006246715562672268/1008709804916752394)
[Червяк в буре](https://discord.com/channels/617003227182792704/1006109543429832804/1008709186860875838) (fixes https://github.com/ss220-space/Paradise/issues/1157)
[Дюпаем абилки](https://discord.com/channels/617003227182792704/1008614641934540822/1008709393099006142)
[Хитрые неоткрываемые сейфы](https://discord.com/channels/617003227182792704/617004034405957642/1008928131597480096)



## Why It's Good For The Game
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
Правка багов и уборка за другими

## Images of changes
<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif of your feature if you want -->

## Changelog
:cl:
fix: Проверка борера для бури
fix: Корректный символ для эмоута Танцевать и Стонать
fix: Исправление ошибок лодаута
fix: Корректировка преф-вербов и Game Preferences
fix: Возвращение стандартных цветов в examine
fix: Предотвращение получения нескольких заклинаний трансформа в дракона с лавы
fix: Отсутсвтующие диеты у Киданов и Драсков
fix: Убрана возможность дюпа абилок вринов, унатхов, теней и слаймочелов
fix: Убран свет при смене расы с нуклеации
fix: Убрано вечное оповещение о свете при смене расы с теней
fix: Открытие сейфов через AltClick
feat: Перенос interaction-эмоутов в кастом-эмоут прок
feat: Тентакля генокрада теперь гарантирует притягивание (мувделей на 1 секунду при попадании в режиме не дизарм)
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

larentoun committed:
feat: freeze mob for a 1 sec on grabbing intent for tentacle (#1031)

larentoun committed:
fix: Корректный символ для эмоута

larentoun committed:
fix: Проверка иммунитета борера от бури

larentoun committed:
fix: Исправление ошибок

Update code/modules/client/preference/loadout/loadout_suit.dm

Co-authored-by: Aziz Chynaliev <azizonkg@gmail.com>

larentoun committed:
fix: Корректировка преф-вербов и Game Preferences

Bizzonium commented on code/modules/client/preference/loadout/loadout_suit.dm:
```diff
@@ -108,6 +108,10 @@
 	allowed_roles = list("Head of Security", "Warden", "Detective", "Security Officer", "Security Pod Pilot")
 
+//SURAGI JACKET
+/datum/gear/suit/suragi_jacket/
```
```suggestion
/datum/gear/suit/suragi_jacket
```

larentoun committed:
fix: examine color

larentoun committed:
feat: interaction с куклой переведен в эмоуты (поддержка рунчата)

Bizzonium commented:
Если хочешь ребейз, то нужно добавить к каждому коммиту ссылку на PR

larentoun commented:
> Если хочешь ребейз, то нужно добавить к каждому коммиту ссылку на PR

Когда делаю ребейз, то коммитам в ребейзе нужно ссылку на ПР делать? В плане, в заголовке коммита?

larentoun committed:
fix: Получение нескольких способностей с зелья дракона

larentoun committed:
fix: Фикс отсутствующих диет

larentoun committed:
fix: Фикс "дюпа" расовых абилок (https://github.com/ss220-space/Paradise/pull/1168)

fix: Фикс "дюпа" расовых абилок (https://github.com/ss220-space/Paradise/pull/1168)

VasilevAY commented:
А не забываешь к issue привязываться?

larentoun commented:
> А не забываешь к issue привязываться?

А у меня доступа нет к кнопкек

VasilevAY commented:
> > А не забываешь к issue привязываться?
> 
> А у меня доступа нет к кнопкек

А у меня тоже нет. Просто в Issue кидай этот ПР, потом придумаем чего. 
вон, как я выше, что в Issue упомянул этот ПР

larentoun commented:
> А у меня тоже нет. Просто в Issue кидай этот ПР, потом придумаем чего.
вон, как я выше, что в Issue упомянул этот ПР

Ну, повторно ПР указала...

VasilevAY commented:
> > А у меня тоже нет. Просто в Issue кидай этот ПР, потом придумаем чего.
> > вон, как я выше, что в Issue упомянул этот ПР
> 
> Ну, повторно ПР указала...

А остальные фиксы?

larentoun commented:
> А остальные фиксы?

Остальное что сама нашла, что в #баг-репорты нашла

VasilevAY commented:
> > А остальные фиксы?
> 
> Остальное что сама нашла, что в #баг-репорты нашла

Прикрепи, тогда, пожалуйста, в дискордике ссылку на пр под теми репортами, который взяла. Немного постараемся почистить нашу разработку и не мусорить

larentoun commented:
> Прикрепи, тогда, пожалуйста, в дискордике ссылку на пр под теми репортами, который взяла. Немного постараемся почистить нашу разработку и не мусорить

[Дюпаем драконов](https://discord.com/channels/617003227182792704/1006246715562672268/1008709804916752394)
[Червяк в буре](https://discord.com/channels/617003227182792704/1006109543429832804/1008709186860875838)
[Дюпаем абилки](https://discord.com/channels/617003227182792704/1008614641934540822/1008709393099006142)

larentoun committed:
fix: Открытие сейфов через AltClick

larentoun commented:
[Хитрые неоткрываемые сейфы](https://discord.com/channels/617003227182792704/617004034405957642/1008928131597480096)

Bizzonium commented:
> > А не забываешь к issue привязываться?
> 
> А у меня доступа нет к кнопкек

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

larentoun commented:
> https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

О, а вот про это не знала, добавила
